components/ee/ws-scheduler/pkg/scaler/driver.go

```diff
@@ -291,7 +292,8 @@ func (wspd *WorkspaceManagerPrescaleDriver) startGhostWorkspaces(ctx context.Con
 				},
 				DeprecatedIdeImage: wspd.Config.IDEImage,
 				IdeImage: &api.IDEImage{
-					WebRef: wspd.Config.IDEImage,
+					WebRef:        wspd.Config.IDEImage,
+					SupervisorRef: wspd.Config.SupervisorImage,
```

Comment:
please add the corresponding entry in the installer

Reply:
Done :heavy_check_mark: